src/quo2/components/numbered_keyboard/numbered_keyboard/style.cljs

```diff
@@ -0,0 +1,18 @@
+(ns quo2.components.numbered-keyboard.numbered-keyboard.style)
+
+(def container
+  {:flex-direction     :row
+   :flex-wrap          :wrap
+   :flex               1
+   :padding-horizontal 34
+   :padding-top        8
+   :padding-bottom     0})
+
+(defn keyboard-item
+  [position]
+  {:width         "33%"
```

Comment:
hmm, I think we try to avoid using fixed percentages like this.
Is it possible to do with flex and margin? I think we should be able to make a grid easily enough here with row and column views

Reply:
Done